### ISSUE TITLE
Attribute 'ignore_label' does not exist in the metadata of dataset '/raid/Bansi/Cityscapes/gtFine/train': metadata is empty.

### ISSUE THREAD

**Issue by bansilol** [bowenc0221/panoptic-deeplab#113]
Hi!
First of all thanks for the great work and making it public :)

I was trying to train the network on the Cityscapes Dataset and I even generated the required labels using the cityscapes scripts. Currently I have the following files: 
-color.png
-instanceIds.png
-labelIds.png
-polgons.jason
-labelTrainIds.png
-cityscapes_panoptic_train/panoptic.png
-cityscapes_panoptic_train.json
-cityscapes_panoptic_train_trainId.json.

When running the train_net.py file with the following command(in the virtual env with all requirements installed): 

python train_net.py --config-file configs/Cityscapes-PanopticSegmentation/panoptic_deeplab_R_52_os16_mg124_poly_90k_bs32_crop_512_1024_dsconv.yaml --num-gpus 2

I get the following error:

Exception has occurred: AttributeError
Attribute 'ignore_label' does not exist in the metadata of dataset '/raid/Bansi/Cityscapes/gtFine/train': metadata is empty.
  File "/home/bansi/detectron2/projects/Panoptic-DeepLab/train_net.py", line 97, in build_train_loader
    mapper = PanopticDeeplabDatasetMapper(cfg, augmentations=build_sem_seg_train_aug(cfg))
  File "/home/bansi/detectron2/projects/Panoptic-DeepLab/train_net.py", line 157, in main
    trainer = Trainer(cfg)
  File "/home/bansi/detectron2/projects/Panoptic-DeepLab/train_net.py", line 171, in <module>
    args=(args,),

I tried pointing the path of all the above mentioned files via the main config file where we have the list of paths for TRAIN but all the paths still show the same error. Can you please let me know if I am missing something.

Thanks in advance :)
